jetstream/core/orchestrator.py

```diff
@@ -508,6 +510,23 @@ def _prefill_thread(self, idx: int):
       del prefill_result
       del request
 
+  def _jax_transfer_prefill_result(self, new_request, target_idx):
```

Comment:
Could we add the typing annotation for the input and outputs?

Reply:
Good point, added.